apps/chat/src/components/Files/FileManagerModal.tsx

```diff
@@ -112,6 +112,16 @@ export const FileManagerModal = ({
     }
     return getExtensionsListForMimeTypes(allowedTypes);
   }, [allowedTypes, t]);
+
+  const typesLabel = useMemo(() => {
+    if (allowedTypesLabel) {
+      return allowedTypesLabel;
+    }
+    if (allowedTypes.length === 1 && allowedTypes[0] === 'image/*') {
+      return t('images');
```

Comment:
Why did you do it only for `image/*`? 
```suggestion
    if (allowedTypes.length === 1 && allowedTypes[0].endsWith('/*') {
      return t(allowedTypes[0].replace('/*', 's'));
```

Reply:
Thank you for the suggestion!
Implemented.